jupiter-tests/src/test/java/org/junit/jupiter/engine/extension/ExtensionRegistryTests.java

```diff
@@ -60,6 +60,7 @@ void newRegistryWithoutParentHasDefaultExtensions() {
 	void newRegistryWithoutParentHasDefaultExtensionsPlusAutodetectedExtensionsLoadedViaServiceLoader() {
 
 		when(configuration.isExtensionAutoDetectionEnabled()).thenReturn(true);
+		when(configuration.createExtensionFilterByPatterns()).thenReturn(__ -> true);
```

Comment:
@marcphilipp 
Based on the comment below, I’ve moved the filter creation codes to the configuration.
- https://github.com/junit-team/junit5/pull/4120#discussion_r1847880228

Accordingly, when mocking, I plan to set it up so that the method returns a lambda that always resolves to `true`. Could you please review this approach?

Reply:
Yes, that looks good. Are there many places that need to be adjusted like that?